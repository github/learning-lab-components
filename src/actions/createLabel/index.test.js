const createLabel = require('./')
const mockContext = require('../../tests/mockContext')

describe('createLabel', () => {
  let context

  beforeEach(() => {
    context = mockContext({}, {
      issues: {
        createLabel: jest.fn()
      }
    })
  })

  it('creates a label', async () => {
    await createLabel(context, { name: 'My label', color: 'f87000' })
    expect(context.github.issues.createLabel).toHaveBeenCalled()
    expect(context.github.issues.createLabel.mock.calls).toMatchSnapshot()
  })
})
