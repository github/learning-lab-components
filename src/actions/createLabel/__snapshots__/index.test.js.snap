// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createLabel creates a label 1`] = `
Array [
  Array [
    Object {
      "color": "f87000",
      "description": undefined,
      "name": "My label",
      "owner": "JasonEtco",
      "repo": "example",
    },
  ],
]
`;
